### DISCUSSION TITLE
How to disable the on/off toggle keyboard shortcut?

### DISCUSSION THREAD

**Discussion by SaltfishAmi** [birchill/10ten-ja-reader#1402] [Q&A] [answered]
![image](https://github.com/birchill/10ten-ja-reader/assets/14184974/f4bcaf95-7318-499d-80c9-994aba2bb467)

This one. Can I disable this feature completely?

**Comment by birtles** [comment, marked as answer]
There's currently no way to disable it from the the 10ten settings interface, however you should be able to disable it by going to `about:addons` then choosing "Manage Extension Shortcuts" from the bottom of the settings menu.

Out of curiousity, is there any reason you want to disable it?

**Reply by SaltfishAmi** [reply]
No. I never use the shortcuts so I disabled them all and out of curiosity I wondered why other ones can be disabled while this one could not.

**Reply by birtles** [reply]
Since I was working on that part of the options screen already I decided to just add the UI for this:

https://github.com/birchill/10ten-ja-reader/assets/1232595/13000fd3-cb7f-4d07-ad5b-bd241a8a3be9
